2nd-gen/packages/tools/swc-tokens/README.md

```diff
@@ -12,6 +12,20 @@ An additional function, `lookupToken()`, is exported for use by `@adobe/postcss-
 - **Resolved value** – A final primitive value
 - **Custom property** – A CSS variable (`var(--swc-gray-500)`)
 
+## Upon Token Data Update
```

Comment:
This seemed like the lightest-weight way to enable this across the affected packages, but we can evolve this process down the line if needed.

This should _not_ need to be run often, as token values don't change frequently.